lib/core/src/lib/comments/comments.component.scss

```
@@ -12,10 +12,18 @@ adf-comments {
 
             #{$mat-form-field} {
                 width: 100%;
-            }
 
-            #{$mat-form-field-subscript-wrapper} {
-                display: none;
+                &#{$mat-form-field-invalid} {
+                    #{$mat-input-element} {
+                        &::placeholder {
```

I think you don't have to repeat this line to set the color if you use this selector:
```suggestion
                        &::placeholder,
                        &:focus::placeholder {
```